aldor/aldor/src/genc.c

```diff
@@ -2382,13 +2388,79 @@ gc0Prog(Foam ref, Foam foam)
 	gc0AddLine(gcvInitProgCC,  ccoStat(ccoAsst(gccId(ref),
 						   ccoPreAnd(ccoCopy(ccLeft)))));
 
-
-
 	retval = ccoMany2(ccoFDef(gcvSpec, gccProgId(ref), ccParams, ccBody),
 			gc0ListOf(CCO_Many, codeProg));
 	Return(retval);
 }
 
+local CCode
+gc0ProgBody(Foam ref, Foam prog)
+{
+	if (!gc0ProgIsC(prog)) {
+		return gc0ProgBodyOther(ref, prog);
+	}
+	else {
+		return gc0ProgBodyC(ref, prog);
+	}
+}
+
+local CCode
+gc0ProgBodyC(Foam ref, Foam foam)
+{
+	Foam locals  = foam->foamProg.locals;
+	Foam body    = foam->foamProg.body;
+	AInt progFmt = foamProgIndex(foam);
+
+	CCode ccBody  = ccoCompound(gc0Compound(locals, body, ref, progFmt,
+					       gcvIsLeaf));
+	return ccBody;
+}
+
+local CCode
+gc0ProgBodyOther(Foam ref, Foam prog)
+{
+	return ccoCompound(ccoStat(gcFiHalt(ccoIdOf("100"))));
+}
+
+local Bool
+gc0ProgIsC(Foam foam)
+{
+	int i;
```

Comment:
Unused variable (because C99 is used further down). Are we using C99 now?

Reply:
Yes.. I think it's probably time to switch?

Reply:
Yes, I agree, this syntax is now also supported by MSVC (since 2015 I think), so there is no real reason to keep this anymore.